src/ResourceManager/Network/ChangeLog.md

```diff
@@ -22,6 +22,10 @@
 
 ## Version 6.2.0
 * Enable Traffic Analytics parameters on Network Watcher cmdlets
+* Updated below cmdlets for Application Gateway
```

Comment:
@akshaysngupta please move the snippet you are adding here to underneath the `## Current Release` header above

Reply:
corrected.